tests/us_reverse_geo/test_Client.js

```diff
@@ -0,0 +1,30 @@
+const chai = require("chai");
+const chaiAsPromised = require("chai-as-promised");
+chai.use(chaiAsPromised);
+const expect = chai.expect;
+const Client = require("../../src/us_reverse_geo/Client");
+const Lookup = require("../../src/us_reverse_geo/Lookup");
+const MockSender = require("../fixtures/mock_senders").MockSender;
+
+describe("A US Reverse Geo client", function () {
+	it("has an inner sender.", function () {
+		let mockSender = new MockSender();
+		let client = new Client(mockSender);
+
+		expect(client.sender).to.deep.equal(mockSender);
+	});
+
+	it("builds a request for a single lookup with the correct request parameters.", function () {
+		let mockSender = new MockSender();
```

Comment:
I think you will need to rework this due to some errors being unhandled when the tests are run.

```
UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'then' of undefined
    at /Users/cami/dev/src/github.com/smartystreets/smartystreets-javascript-sdk/src/us_reverse_geo/Client.js:24:5
```

This should be 
```suggestion
		let mockSender = new MockSenderWithResponse();
```
and pass in the `expectedPayload`. 

Or if you don't want to have to mess with the `expectedPayload`, then you could create a new MockSender that implements the `this.send` method but returns a promise like `MockSenderWithResponse` does so we don't get the `UnhandledPromiseRejectionWarning` that I pasted in above